app/views/health/patient/_remaining_to_engage.haml

```diff
@@ -24,15 +25,16 @@
           Participation Form
 
   %li
-    - # Release forms are valid for two years (if they occur after the enrollment date)
+    - # Release forms are valid if the signature is within 1 year prior to enrollment start date and are valid
+    - # for two years from the signature date
     - release = patient.release_forms.active.recent.during_current_enrollment.first
```

Comment:
Doesn't `during_current_enrollment` mess with the 1 year check on line 31?

Reply:
`during_current_enrollment` was checking the contributing referrals... I have added `during_contributing_enrollments` and moved these to that... If we discover code that should be considering auto re-enrollments, they will need to move to this new scope.